src/components/TextInput/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InpuText, LabelText } from './styles';
+const TextInput = ({ label, id, type, logical, placeholder }) => {
+  return (
+    <>
+      <LabelText htmlFor={id}>{label}</LabelText>
+      <br />
+      <InpuText type={type} id={id} {...logical} placeholder={placeholder} />
```

Comment:
As we are now using `label` there's no need for `placeholder` you can delete it.

Reply:
All ways need a placeholder, principal to show Format for the input, The propose of component is to be A general to generate another. if no need a placeholder doesn't use the props